src/redux/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,58 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const baseUrl = 'https://api.spacexdata.com/v3/rockets';
+
+const initialState = {
+  allRockets: [],
+  isLoading: false,
+};
+
+export const getRockets = createAsyncThunk(
+  'rocket/getRockets',
+  async (name, thunkAPI) => {
+    try {
+      const response = await fetch(baseUrl);
+      const data = await response.json();
+      return data;
```

Comment:
- [ ] Storing the entire data fetched from the API in Redux may result in unnecessary overhead and slow down the application. It would be great to only store the information necessary for this application. Kindly map the incoming `data` to store the required fields suggested in the project requirements. 😎️

**Necessary pieces of data for the app to run**
<img src="https://user-images.githubusercontent.com/17243325/225839648-f34324fc-b70d-40ca-9821-53eae95c22de.png" width="300">